### ISSUE TITLE
Training loss convergence on custom data!

### ISSUE THREAD

**Issue by kadavparth** [ztsrxh/RoadBEV#20]
First of all, thank you for the great work and for making the code available.

I’m currently training the model on a custom dataset of about 1,800 samples consisting of synchronized stereo pairs with LiDAR data. The LiDAR is from a 128-channel Ouster sensor. To improve label quality, I’m applying ICP across the next 15 consecutive frames to densify the point cloud for each frame.

Since IMU data wasn’t available, I’m estimating the ground plane using RANSAC on an ROI-cropped point cloud focused on the road ahead. I then manually adjust the angle per frame to better align the image with respect to the vertical axis (as opposed to the IMU-based alignment used in your setup).

For training, I’m using an NVIDIA RTX Ada 6000 GPU (98GB VRAM), running 200 epochs with a batch size of 8. However, the training loss only decreases to around 1.5 (around epoch 120-ish) and does not improve further until training ends.

I wanted to ask if this is expected behaviour when training on a custom dataset, or if you have any suggestions on what might be limiting performance or how to improve convergence.

Thanks again for your work!
